tests/testthat/testImportSNV.TCGA.R

```diff
@@ -0,0 +1,181 @@
+# testImportSNV.TCGA.R
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+NL <- .PlatformLineBreak()
+# ==== END SETUP AND PREPARE ===================================================
+
+#context("Tools for importing MAF files into a rSNV file")
+options(stringsAsFactors=FALSE)
+
+test_that("parameter errors are correctly handled", {
+    # fMAF is not provided
+    expect_error(importSNV.TCGA())
```

Comment:
```expect_error()``` uses a regex as a second argument, better use that. Otherwise you can't tell if the error you get is the error you expect - eg. whether it comes from the system, or from ```.checkArgs()```